src/components/users/subscription-details.tsx

```diff
@@ -44,13 +45,18 @@ const SubscriptionDetails: React.FunctionComponent<SubscriptionDetailsProps> = (
   }, [stripeCustomerId, slug])
 
   const subscriptionName = subscriptionData && subscriptionData.product?.name
+  const subscriptionUnitAmount = get(
+    subscriptionData,
+    'latestInvoice.amount_due',
+    subscriptionData?.price?.unit_amount,
+  )
```

Comment:
Getting the price to show from the latest invoice feels like a band-aid solution.